### DISCUSSION TITLE
可否在Issue内增加类似good first issue的tag？

### DISCUSSION THREAD

**Discussion by JerryChan31** [tusen-ai/naive-ui#1588] [Ideas]
可否在Issue内增加类似good first issue的tag？方便新的开源参与者加入。

**Comment by 07akioni** [comment]
Good idea

**Reply by 07akioni** [reply]
已经增加了

你可以加一下钉钉群，然后在群里 at 群主，我把你拉进开发者钉钉群

**Reply by JerryChan31** [reply]
![image](https://user-images.githubusercontent.com/14790315/142188253-cda7dba4-f884-408e-ae96-74c194664a72.png)

这个是我🤣

**Comment by Talljack** [comment]
已经将 good first issue 改成了 pr welcome 的 label，这些是比较容易修改的 issue，欢迎提交 pr
